spec/std/enumerable_spec.cr

```diff
@@ -262,7 +262,7 @@ describe "Enumerable" do
   describe "in_groups_of" do
     assert { [1, 2, 3].in_groups_of(1).should eq([[1], [2], [3]]) }
     assert { [1, 2, 3].in_groups_of(2).should eq([[1, 2], [3, nil]]) }
-    assert { ([] of Int32).in_groups_of(2).should eq([] of Array(Array(Int32 | Nil))) }
+    assert { ([] of Int32).in_groups_of(2).should eq([] of Array(Int32 | Nil)) }
```

Comment:
Ok I can see the test is wrong. However I think the behaviour of comparison of arrays of different type was intentional.